packages/components/src/button/README.md

```diff
@@ -1,29 +1,126 @@
 # Button
+Buttons let users take actions and make choices with a single click or tap.
 
-Buttons express what action will occur when the user clicks or taps it. Buttons are used to trigger an action, and they can be used for any type of action, including navigation.
+![Button components](https://make.wordpress.org/design/files/2019/03/button.png)
 
-The presence of a `href` prop determines whether an `anchor` element is rendered instead of a `button`.
+## Table of contents
+
+1. [Design guidelines](http://#design-guidelines)
+2. [Development guidelines](http://#development-guidelines)
+3. [Related components](http://#related-components)
+
+## Design guidelines
+
+### Usage
+
+Buttons tell users what actions they can take and give them a way to interact with the interface. You’ll find them throughout a UI, particularly in places like:
+
+- Modals
+- Forms
+- Toolbars
+
+### Best practices
```

Comment:
Just noting that this information is here too, not sure if it's a problem that they're duplicated.

https://wordpress.org/gutenberg/handbook/designers-developers/developers/components/button-group/#best-practices

Reply:
Those are for a different component — the ButtonGroup. They share some of the same guidelines, but are a little different.

Reply:
That link is for a different component, a ButtonGroup. It shares similarities with many of the rules here, but it's a bit different.